carsus/io/cmfgen/util.py

```diff
@@ -270,7 +270,7 @@ def get_opproject_phixs_table(threshold_energy_ryd, a, b, c, d, e, n_points=1000
     return phixs_table
 
 
-def get_hummer_phixs_table(threshold_energy_ryd, a, b, c, d, e, f, g, h, n_points=1000):
```

Comment:
We were aware that the model in the paper has 8 parameters but for some reason, the function here is using just 7. I don't know if we should remove the unused variable or not.